optimum/habana/diffusers/pipelines/ddpm/pipeline_ddpm.py

```diff
@@ -180,5 +180,5 @@ def __call__(
         if not return_dict:
             return (image,)
 
-        throughput = (end_time - start_time) / batch_size
```

Comment:
can you check actually what's the sample size? Seems it's running 1000 itereation with batch16, then it should be 16*1000 sample.

Reply:
> can you check actually what's the sample size? Seems it's running 1000 itereation with batch16, then it should be 16*1000 sample.

The diffusion model runs for 1000 inference steps per batch (16), It process the same 16 samples for 1000 iterations. So the sample size for throughput should still be 16.

Reply:
> Also if you look at other examples (pipeline controlnet, or pipeline stable diffusion), they also remove warmup time (first 2-3 steps) since compile time takes much longer in HPU for first 2-3 iteration. If that's something quick to add, it would be good to also add it here. throughput_warmup_steps

Sure I could add that, will update the PR with the change requested.

Reply:
@jiminha 
Updated throughput calculation to ignore warmup time, this can be configured with a command line argument --throughput_warmup_steps. The throughput is now displayed similar to other diffusion models with samples_per_second as shown below,

 Speed metrics: {'generation_runtime': 105.8193, 'generation_samples_per_second': 0.151, 'generation_steps_per_second': 151.201}